nvflare/fuel/flare_api/api_spec.py

```diff
@@ -205,6 +227,256 @@ def delete_job(self, job_id: str):
 
     @abstractmethod
     def get_system_info(self) -> SystemInfo:
+        """Get general info of the FLARE system"""
+        pass
+
+    def get_client_job_status(self, client_names: List[str] = None) -> List[dict]:
```

Comment:
Since this is a "spec", or "abstract class".
It might be better if we add `@abstractmethod` to these method definitions
SO we enforced whoever inherits this class need to implement these methods.

Reply:
yes

Reply:
done as suggested